amit/html/Assigment4/Amit4_3.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document1</title>
+    </head>
+    <style>
+        th,
+        td {
+            border-top: 1px solid black;
+            border-left: 1px solid black;
+            border-bottom: 1px solid gainsboro;
+            border-right: 1px solid gainsboro;
+            padding: 5px;
+            border-width: 2px;
+        }
+        .a {
+            border-top: 1px solid black;
+            border-left: 1px solid black;
+            border-bottom: 1px solid black;
+        }
+        .b {
+            background-color: cornsilk;
+        }
+        .c {
+            background-color: rgb(79, 79, 192);
+        }
+        .d {
+            background-color: cyan;
+        }
+    </style>
+    <body>
+        <table style="width: 50%" border="2" cellpadding="23">
+            <tr>
+                <th rowspan="3">DAY</th>
+                <th colspan="5">seminar</th>
+            </tr>
+            <tr>
+                <th colspan="2">schedule</th>
+                <th colspan="3" ; rowspan="2">Topic</th>
+            </tr>
+            <tr>
+                <th>Begin</th>
+                <th>End</th>
+            </tr>
+            <tr>
+                <td rowspan="2">Monday</td>
+                <td rowspan="2" ; class="b">8:00 a.m.</td>
+                <td rowspan="2" ; class="c">5:00 p.m.</td>
```

Comment:
remove space before ;

Reply:
this is also after prettier happen i think its prettier format